common/api/appui-react.api.md

```diff
@@ -2075,40 +2079,64 @@ export interface FrameworkToolSettings {
     useDefaultToolSettingsProvider: boolean;
 }
 
-// @public
+// @public @deprecated
 export class FrameworkUiAdmin extends UiAdmin {
+    // @deprecated
     closeDialog(dialogId: string): boolean;
+    // @deprecated
     closeToolSettingsPopup(): boolean;
+    // @deprecated
     get cursorPosition(): XAndY;
-    // (undocumented)
-    getKeyins(): KeyinEntry[];
+    // @deprecated (undocumented)
+    getKeyins(): Keyin.KeyinEntry[];
+    // @deprecated
     hideCalculator(): boolean;
+    // @deprecated
     hideCard(): boolean;
+    // @deprecated
     hideHTMLElement(): boolean;
+    // @deprecated
     hideInputEditor(): boolean;
+    // @deprecated
     hideKeyinPalette(): boolean;
+    // @deprecated
     hideMenuButton(id: string): boolean;
+    // @deprecated
     hideToolbar(): boolean;
+    // @deprecated
     get isFocusOnHome(): boolean;
-    // (undocumented)
-    get localizedKeyinPreference(): KeyinFieldLocalization;
-    set localizedKeyinPreference(preference: KeyinFieldLocalization);
-    // @internal (undocumented)
-    onInitialized(): void;
-    openDialog(uiDataProvider: DialogLayoutDataProvider, title: string, isModal: boolean, id: string, optionalProps?: DialogProps_2): boolean;
+    // @deprecated (undocumented)
+    get localizedKeyinPreference(): Keyin.KeyinFieldLocalization;
```

Comment:
Hm... it also doesn't complain with non-enum types and if the name matches and the [type is in another namespace instead of global](https://www.typescriptlang.org/play?#code/KYOwrgtgBAKsDOAXKBvAUFTUAaUC8UADADQZYCa+UAjGgL5pqICeADsLG8NVellAEMAXFCQAnAJYgA5mUwAjEeAjzgY+o1CRYCRDz5ZcBEnKiUCtBmmAAPVgHsxyEAIgJWAgMYcAaqlO2Dk5QLOyc7DwEBvyYwqKIkjKm-IpQyqpimKYMVoGOzq7uXhww-vx5wVrQcEj6yYZUJjEUVNSk-FbldvlQVTpIZc04je1D5jSjzQD0U1AAWlQATMRQM1AA8gCqAMIAElAA7gAWoFBgIJ72EG4giMAAJtmMXUHIoSVckYPNceJSskMoKl0mosh0NIw0AAzc6eRASewgEK6AAUNhENUQAEpvlBLiB4PYADbAAB0RPs0jRWIA3BCYRd4YjkUhFmiRD5STBPljonjEYSSeTKdTTGJgIgwGIkTZ6HTGHckCjCLSmKjMaTsKrFYgUZjqJraatZls9mqlTAubpDTTjVAiWBPABrZgAfnNust+ptdtNu0Y+IGiCO-wxn14pjiACIQ1HJkCRIsIYGCm54B5vDAQzIMVzw1FIyIo8x7HHTKkAMwQknIcXwXPciJ0mtQOuLMNNtB1qg6tkuNMZ4BZ-5YrsIRY93R9wrp4rDmSjoA): 

Willing to be strict, but perhaps this is okay? To answer the below question of why, I wanted to move the type outside of `FrameworkUiAdmin`, keeping the same name, but allowing users to migrate to the non-deprecated version. Without putting it in a namespace, we'd have conflicts exporting from the barrel file. 

I suppose I could export the new version but not in the barrel (and remove the namespace). Then, revert to using the old deprecated type for any public apis.

Reply:
Ah ha, right! Thanks

Reply:
Done